### PR TITLE
To-do-list-review

### PR DESCRIPTION
**Pull request by eudondian**
Pulled for review
Edited the head section of HTML file

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="./styles/correct.css"> -->
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v3.0.1/css/line.css">
```
Comment (Ney2) [inline src/index.html]: Kindly remove the unwanted line of code to enhance the readability of your code

**Review by Ney2 — CHANGES_REQUESTED** [review]
Hi @eudondian,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



To highlight 
    - No Linter errors :white_check_mark: 
    - Follow Github flow :white_check_mark: 

Required Changes ♻️ 
     - Kindly correct the title of your project from Documents to the one that describes your work.

- [ ] _Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/index.html (hunk 1 above)
Comment (eudondian) [reply src/index.html]: image

**Review by Ney2 — APPROVED** [review]
Hi @eudondian,

## Status: Approved ✔️ 🥇 
Your project is complete! There is nothing else to say other than... it's time to merge it.
Congratulations! 🎉



Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding 👏 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Pull request merged**